README.md

```diff
@@ -1,58 +1,160 @@
+docker image ls
+// REPOSITORY                 TAG       IMAGE ID       CREATED             SIZE
+// f-oh-data                  latest    f01eda341ff6   15 minutes ago      1.01GB
+// <none>                     <none>    a08ee5a22bcb   About an hour ago   1.01GB
+// <none>                     <none>    1f7695414f4c   21 hours ago        1.01GB
+// <none>                     <none>    cae571a5cf0b   22 hours ago        1.01GB
+// <none>                     <none>    7b36d96948f8   22 hours ago        1.01GB
+// gowokegobroke/oniro-data   latest    490f1e49f9da   6 months ago        1.17GB
+```
+Here we see the name of the `image` we need to find, run it first, then find the container id in the container and execute the bash command  
+
+```typescript
+//Run the desired server
+docker run -d -p 5000:5000 f-oh-data
+
+//View the id of the running server in the container
+docker container ls
+
+//After finding the id, replace the following id
+docker exec -it 70cb78e186c8 bash
+```
+Now we have found the file path of the target  
+`data\` path: `/app/data/`
+`appAppList.json` path: `/app/allAppList.json`  
+
+Remember these paths, you will need to update the `Dockerfile` file later  
+
+Use the `scp` command to transfer the local app's hap file to your own server file  
+```typescript
+scp C:\Users\c84381641\onBoarding\repos\appStore\f-oh-data\data\com.example.oniromathpunching\icon.png ssh [username]@[user ip]:f-oh-data/data/com.example.oniromathpunching
+
+C:\Users\c84381641\onBoarding\repos\appStore\f-oh-data\data\com.example.oniromathpunching\oniromathpunchingv1.0.0.hap ssh [username]@[user ip]:f-oh-data/data/com.example.oniromathpunching
+```
+> First, create the directory `data/com.example.oniromathpunching` in your server  
+
+You also need to add the relevant configuration information of your application in `allAppList.json` as follows(I use math punching app as example):  
+```typescript
+{
+    "id": 29,
+    "name": "Math Punching",
+    "desc": "A learning assistant help with math",
+    "icon": "/data/com.example.oniromathpunching/icon.png",
+    "vender": "Chen",
+    "packageName": "com.example.oniromathpunching",
+    "version": "1.0.0",
+    "hapUrl": "/data/com.example.oniromathpunching/oniromathpunching-v1.0.0.hap",
+    "type": "app",
+    "tags": "Utility，Study",
+    "openSourceAddress": "https://github.com/imansmallapple/app-math_practice_demo",
+    "releaseTime": "2024-11-12"
+}
+```
+
+After uploading, update the Dockerfile file command through vi  
+```typescript
+vi Dockerfile
+```
+Enter `i` to enter `insert` mode, then add the following command  
+```typescript
+COPY allAppList.json /app/allAppList.json
+COPY data/ /app/data/
+```
+Finally, shut down the image in the container by using docker stop id  
+
+First `build` update the `f-oh-data` image file  
+```typescript
+docker build -t f-oh-data .
+```
+
+Re-run the `f-oh-data` image file  
+```typescript
+docker run -d -p 5000:5000 f-oh-data
+```
+
+problem solved  
```

Comment:
these instructions are not relevant for this readme. These are enough.

````suggestion
## Start the server with Docker
### **Build the Docker Image**
Run the following command:

```bash
docker build -t f-oh-data .
```

### **Run the Docker Container**
To start the container:

```bash
docker run -d -p 5000:5000 f-oh-data
```
````